drivers/at24cxxx/mtd/mtd.c

```diff
@@ -78,6 +83,7 @@ static int _mtd_at24cxxx_power(mtd_dev_t *mtd, enum mtd_power_state power)
 const mtd_desc_t mtd_at24cxxx_driver = {
     .init = _mtd_at24cxxx_init,
     .read = _mtd_at24cxxx_read,
```

Comment:
You can now also drop the `.read` function.
Same is true for `.write`, not sure why it's even still there.

Reply:
I removed them. So `read` and `write` are considered legacy / deprecated? If this is the case it should be mentioned in the docstring I think.